### ISSUE TITLE
Exchange shared events between different users, and assign these shared events to all the users involved

### ISSUE THREAD

**Issue by Flechman** [Sosol02/sdp#123]

**Comment by Flechman** [comment]
This issue is postponed untill next sprint as other more important issues needed to be addressed before this one.

**Comment by Flechman** [comment]
After tackling the problem of how to exchange info beteen users, i think it will take a bit more time than expected.

**Comment by Flechman** [comment]
Considering the time left for developping the app, this feature will be implemented only after a first version of the app is done.
